content/doc/book/managing/system-properties.adoc

```diff
@@ -519,96 +609,115 @@ The following placeholders are supported for this value:
 
+[[jenkins_model_Jenkins_slaveAgentPortEnforce]]
 `jenkins.model.Jenkins.slaveAgentPortEnforce`::
     **Default:** `false` +
     **Since:** 2.19.4 and 2.24 +
     **Description:** If true, enforces the specified `+jenkins.model.Jenkins.slaveAgentPort+` on startup and will not allow changing it through the UI
 
+[[jenkins_model_Jenkins_workspacesDir]]
 `jenkins.model.Jenkins.workspacesDir`::
     **Default:** $\{JENKINS_HOME}/workspace/$\{ITEM_FULL_NAME} +
     **Since:** 2.119 +
     **Description:** Allows to change the directory layout for the job workspaces on the master node. See `+jenkins.model.Jenkins.buildsDir+` for supported placeholders.
 
+[[jenkins_model_JenkinsLocationConfiguration_disableUrlValidation]]
 `jenkins.model.JenkinsLocationConfiguration.disableUrlValidation`::
     **Default:** `false` +
     **Since:** 2.197 / LTS 2.176.4 +
     **Description:** Disable URL validation intended to prevent an XSS vulnerability. See link:/security/advisory/2019-09-25/#SECURITY-1471[SECURITY-1471] for details.
 
+[[jenkins_model_StandardArtifactManager_disableTrafficCompression]]
 `jenkins.model.StandardArtifactManager.disableTrafficCompression`::
     **Default:** `false` +
     **Since:** 2.196 +
     **Description:** `+true+` to disable GZIP compression of artifacts when they're transferred from slave nodes to master.  Uses less CPU at the cost of increased network traffic.
 
-`jenkins.security.ApiTokenProperty.adminCanGenerateNewTokens `::
+[[jenkins_security_ApiTokenProperty_adminCanGenerateNewTokens]]
+`jenkins.security.ApiTokenProperty.adminCanGenerateNewTokens`::
     **Default:** `false` +
     **Since:** 2.129 +
     **Description:** `+true+` to allow users with `+ADMINISTER+` permission to create API tokens using the new system for any user. Note that the user will not be able to use that token since it's only displayed to the creator, once.
 
+[[jenkins_security_ApiTokenProperty_showTokenToAdmins]]
 `jenkins.security.ApiTokenProperty.showTokenToAdmins`::
     **Default:** `false` +
     **Since:** 1.638 +
     **Description:** True to show API tokens for users to administrators on the user configuration page. This was set to `false` as part of https://wiki.jenkins-ci.org/display/SECURITY/Jenkins+Security+Advisory+2015-11-11#JenkinsSecurityAdvisory2015-11-11-APItokensofotherusersavailabletoadmins[SECURITY-200]
 
+[[jenkins_security_FrameOptionsPageDecorator_enabled]]
 `jenkins.security.FrameOptionsPageDecorator.enabled`::
     **Default:** `true` +
     **Since:** 1.581 +
     **Description:** Whether to send `+X-Frame-Options: sameorigin+` header, set to `false` to disable and make Jenkins embeddable
 
-`jenkins.security.stapler.StaplerDispatchValidator.disabled`::
+[[jenkins_security_stapler_StaplerDispatchValidator.disabled_disabled]]
+`jenkins.security.stapler.StaplerDispatchValidator.disabled.disabled`::
```

Comment:
```suggestion
[[jenkins_security_stapler_StaplerDispatchValidator_disabled]]
`jenkins.security.stapler.StaplerDispatchValidator.disabled`::
```

Reply:
Thank you @MarkEWaite for the meticulous review. I'm sorry I didn't catch those myself. I will make the corrections.

Reply:
Corrections pushed.